spec/ref_arch_setup/bolt_helper_spec.rb

```diff
@@ -130,10 +131,31 @@
       "#{RefArchSetup::RAS_MODULE_PATH} --nodes #{nodes}"
     end
 
+    context "when a modulepath is specified" do
+      expected_output = "All Good"
+      expected_status = 0
+      modulepath = "./my_modules"
+
+      it "uses the specified value" do
+        expected_command = "bolt task run #{task} VAR1=1 VAR2=2 --modulepath "\
```

Comment:
I have noticed this var sometimes has @ and sometimes doesn't... that seems suspect.  Are you sure that is what you want to do?

Reply:
'expected_command' is just used for the cases where the modulepath is specified. I've updated it to use '@expected_command' since it won't impact the other cases and added a comment to explain.